src/backend/src/prisma-query-args/work-package-template.query-args.ts

```diff
@@ -0,0 +1,11 @@
+import { Prisma } from '@prisma/client';
+
+export const workPackageTemplateQueryArgs = Prisma.validator<Prisma.Work_Package_TemplateArgs>()({
+  include: {
+    userCreated: true,
+    userDeleted: true,
+    blockedBy: true
+  }
+});
+
+export const blockedByInfoQueryArgs = Prisma.validator<Prisma.Blocked_By_InfoArgs>()({});
```

Comment:
if this is going to be empty anyways there's no need for it, you can just delete this entire query args